vendor/bat-native-ads/src/bat/ads/internal/notifications.cc

```diff
@@ -49,33 +49,42 @@ void Notifications::Initialize(InitializeCallback callback) {
 bool Notifications::Get(const std::string& id, NotificationInfo* info) const {
   DCHECK(is_initialized_);
 
-  auto notification = notifications_.find(id);
-  if (notification == notifications_.end()) {
+  auto iter = std::find_if(notifications_.begin(), notifications_.end(),
+      [&id](const auto & v){return v.id == id;});
+  if (iter == notifications_.end()) {
     return false;
   }
 
-  *info = notification->second;
+  *info = *iter;
 
   return true;
 }
 
 void Notifications::Add(const NotificationInfo& info) {
   DCHECK(is_initialized_);
 
-  notifications_.insert({info.id, info});
+  notifications_.push_back(info);
 
   SaveState();
 }
 
+void Notifications::Remove() {
+  if (!notifications_.empty()) {
+    notifications_.pop_front();
+  }
+}
+
 bool Notifications::Remove(const std::string& id) {
   DCHECK(is_initialized_);
 
-  if (!Exists(id)) {
+  auto iter = std::find_if(notifications_.begin(), notifications_.end(),
+      [&id](const auto & v){return v.id == id;});
```

Comment:
CN we please change v to notification

Reply:
Yes, we can definitely can change `v` to `notification` if you want. But `v` is abbreviation of `value` and used only in context of this short lambda function. What's wrong with it?

Reply:
Consistency as in ads and ledger libs we do not use abbreviations, as it makes the code more readable. So we prefer variables which refer to what they are.

Reply:
done